pkg/apis/ibmcloudprovider/v1beta1/ibmcloudproviderconfig_types.go

```diff
@@ -42,20 +42,22 @@ type IBMCloudMachineProviderSpec struct {
 
 	// Actuator will apply these tags to an virtual server instance if not present in additon
 	// to default tags applied by the actuator
-	// TODO
-	// Tags []string `json:"tags,omitempty"`
+	Tags []TagSpecs `json:"tags,omitempty"`
 
 	// TODO: Add labels to the virtual server
 	// Labels map[string]string `json:"labels,omitempty"`
 
 	// Image is the id of the custom OS image in VPC
-	// Example: a123-ade15224-ed7y-5e44-ye56-8199d4557699
+	// Example: rchos-4-4-7 (Image name)
 	Image string `json:"image"`
 
 	// Profile indicates the flavor of instance.
 	// Example: bx2-8x32 (8 vCPUs, 32 GB RAM)
 	Profile string `json:"profile"`
 
+	// Region of the virtual machine
+	Region string `json:"region"`
```

Comment:
Same question here about whether or not it would be valid to create a Machine in a region different from the cluster region.

Reply:
Would this field even be needed in the case that it were invalid?

Reply:
Good point -- Conceptually It should be possible to have worker nodes in different `Regions` but not sure how that is managed inside an Openshift cluster. 

- Can we have more than 1 region in Infrastructure CR -- I believe not.
- Can we create a new Infrastructure CR with a different region -- This may not work because most CO's are looking at `name: cluster`` infrastructures.config.openshift.io` resource. 

I tried to stay consistent with other Cloud provider specs as well, so we could leave it here.

Reply:
@elmiko Thoughts?

Reply:
iirc, we can have machines in different regions but they are grouped at the machineset level. in terms of the this provider coinfig though, i think it's perfectly valid to have the region in here, you can see a similar example in `placement` of the aws provider https://github.com/openshift/cluster-api-provider-aws/blob/master/pkg/apis/awsprovider/v1beta1/awsproviderconfig_types.go#L78